README.md

```diff
@@ -7,7 +7,7 @@ Displays CPU usage (%) & Memory usage (bytes) for specific processes. Uses regul
 
 |     OS    | Linux | Windows | SmartOS | OS X |
 |:----------|:-----:|:-------:|:-------:|:----:|
-| Supported |   v   |    -    |    -    |  -   |
+| Supported |   v   |    v    |    v    |  v   |
```

Comment:
Lets not add SmartOS here? since, we cant test it... or do we assume it will work?